### PR TITLE
Mandatory Batch No and Expiry Date and defulat todate set in Manufacturing Date in purchase Receipt item table.

### PR DESCRIPTION
**Pull request by PITPL-Gayatri-Gajjjar**

### CODE REVIEW

File: erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json [hunk 1]
```diff
@@ -1,1070 +1,1074 @@
...
-    "modified": "2022-08-01 04:00:32.435383",
-    "modified_by": "Administrator",
-    "module": "Stock",
-    "name": "Purchase Receipt Item",
-    "owner": "Administrator",
-    "permissions": [],
-    "quick_entry": 1,
-    "sort_field": "modified",
-    "sort_order": "DESC"
+ "actions": [],
```
Comment (PITPL-Kaushal-Khute) [inline erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json]: remove indention change from json file.

**Review by PITPL-Kaushal-Khute — CHANGES_REQUESTED** [review]
remove indention change from json file.

**Review by PITPL-Kaushal-Khute — CHANGES_REQUESTED** [review]
remove indention change from json file.

File: erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json (hunk 1 above)
Comment (PITPL-Gayatri-Gajjjar) [reply erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json]: Done

**Pull request closed without merging**